### PR TITLE
[WebAPI] TC dev for Automation Support for Chromium 52

### PR DESCRIPTION
**Pull request by renchenglei**
Impacted tests(approved): new 5, update 0, delete 0
Unit test platform: Crosswalk for Android 22.52.557.0
Unit test result summary: pass 5, fail 0, block 0

https://crosswalk-project.org/jira/browse/CTS-1852

### CODE REVIEW

**Comment by renchenglei** [comment]
@Honry PTAL, Thanks!

File: webapi/tct-webaudio-w3c-tests/webaudio/automation-support.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        Ren, Chenglei <chenglei.ren@intel.com>
+-->
+
+<html>
+  <head>
+    <title>WebAudio Test: dynamicscompressornode_reduction_default_value</title>
+    <link rel="author" title="Intel" href="http://www.intel.com" />
+    <link rel="help" href="https://webaudio.github.io/web-audio-api" />
+    <meta name="flags" content="" />
+    <meta name="assert" content="Check if DynamicsCompressorNode.reduction default value is '0'" />
```
Comment (Honry) [inline webapi/tct-webaudio-w3c-tests/webaudio/automation-support.html]: Incorrect purpose.

File: webapi/tct-webaudio-w3c-tests/webaudio/automation-support.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+Authors:
+        Ren, Chenglei <chenglei.ren@intel.com>
+-->
+
+<html>
+  <head>
+    <title>WebAudio Test: dynamicscompressornode_reduction_default_value</title>
```
Comment (Honry) [inline webapi/tct-webaudio-w3c-tests/webaudio/automation-support.html]: Incorrect title.

File: webapi/tct-webaudio-w3c-tests/webaudio/automation-support.html [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+Authors:
+        Ren, Chenglei <chenglei.ren@intel.com>
+-->
+
+<html>
+  <head>
```
Comment (Honry) [inline webapi/tct-webaudio-w3c-tests/webaudio/automation-support.html]: Please follow the latest template, https://github.com/crosswalk-project/crosswalk-test-suite/blob/master/tools/template/script_test_case.html

**Comment by renchenglei** [comment]
@Honry I have modified the code, ptal, thanks!

File: webapi/tct-webaudio-w3c-tests/webaudio/automation-support.html [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+<meta charset="utf-8">
+<title>Web Audio: dynamicscompressornode_reduction_default_value</title>
```
Comment (Honry) [inline webapi/tct-webaudio-w3c-tests/webaudio/automation-support.html]: Still incorrect title, this test is for Automation support.

**Comment by Honry** [comment]
LGTM

**Pull request merged**